projects/storefrontlib/cms-components/navigation/category-navigation/category-navigation.component.html

```diff
@@ -1,8 +1,8 @@
 <cx-navigation-ui
   *ngIf="data$ | async as data"
   [node]="node$ | async"
-  [ngClass]="data.styleClass"
-  [wrapAfter]="data.wrapAfter"
+  [ngClass]="data.styleClass ?? ''"
+  [wrapAfter]="+(data.wrapAfter ?? '')"
```

Comment:
What's the `+` for, converting into a string?

Reply:
convert string to number